Sources/SQLKit/Query/SQLTableIdentifier.swift

```diff
@@ -0,0 +1,15 @@
+public struct SQLTableIdentifier: SQLExpression {
```

Comment:
This doesn't add anything beyond `SQLIdentifier`, so we should just use that directly.